### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by RamonaN**

### CODE REVIEW

File: frontend/src/controllers/login.controller.js [hunk 1]
```diff
@@ -16,11 +16,14 @@
 
                 if (user[0].isAdmin === "true") {
                     $rootScope.isAdmin = true;
+                 
                     $location.path('/adminpage');
                 } else {
                     $rootScope.isAdmin = false;
                     $location.path('/movies');
                 }
+                $scope.user=$rootScope.isAdmin
```
Comment (lightforsin) [inline frontend/src/controllers/login.controller.js]: Is this needed ?

File: frontend/views/movies.html [hunk 2]
```diff
@@ -31,7 +37,7 @@
                 <tbody role="rowgroup">
                     <tr role="row" ng-repeat="movie in movies">
                         <td style="width:auto;">
-                            <p class="movie-title">{{movie.title}}</p>
+                            <p class="movie-title"><img src="../images/{{movie.src}}" style="width:60px;height:50px;float:left;margin-right:20px;">{{movie.title}}</p>
```
Comment (lightforsin) [inline frontend/views/movies.html]: Use ng-src instead of src and it will work

File: frontend/views/partials/modals/moviedetails.view.html [hunk 3]
```diff
@@ -1,37 +1,37 @@
 
-  <div class="modal fade" id="detailModal" tabindex="-1" role="dialog">
+   <div class="modal fade" id="detailModal" tabindex="-1" role="dialog">
       <div class="modal-dialog">
         <div class="modal-content">
     
-          <!-- Modal Header -->
+          
           <div class="modal-header">
             <img src="../images/john-wick-chapter-2-poster.jpg" class="mx-auto" style="width:auto;height: 150px">
```
Comment (lightforsin) [inline frontend/views/partials/modals/moviedetails.view.html]: use ng-src

File: frontend/views/partials/modals/moviedetails.view.html [hunk 4]
```diff
@@ -1,37 +1,37 @@
 
-  <div class="modal fade" id="detailModal" tabindex="-1" role="dialog">
+   <div class="modal fade" id="detailModal" tabindex="-1" role="dialog">
```
Comment (lightforsin) [inline frontend/views/partials/modals/moviedetails.view.html]: Do we still need this file ? We are using moviedetails.html, right ?

File: frontend/views/partials/modals/moviedetails.html [hunk 5]
```diff
@@ -1,14 +1,14 @@
-<div>
+<div class="styling">
     <div>
-        <div>
+        <div class="row">
             <!-- Modal Header -->
-            <div>
-                <img src="../images/john-wick-chapter-2-poster.jpg" class="mx-auto" style="width:auto;height: 150px">
+            <div class="col-md-6">
+                <img src="../../../images/{{movie.src}}" class="mx-auto" style="width:100%;height: 100%">
```
Comment (lightforsin) [inline frontend/views/partials/modals/moviedetails.html]: use ng-src instead of src

File: frontend/index.html [hunk 6]
```diff
@@ -25,13 +29,14 @@
...
+        @import url('https://fonts.googleapis.com/css?family=Indie+Flower|Flavors|Archivo+Narrow|Patrick+Hand|Pacifico&display=swap');
         @import url("https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css");
     </style>
+  
 </head>
 
-<body>
-    <div id="page-container">
+<body ng-controller="homeCtrl">
+    <div id="page-container" ng-class="{'body-bg': isCurrentRoute('/movies')}">
```
Comment (lightforsin) [inline frontend/index.html]: I don't think we need to add controllere and logic just to change the background, we can try to do this from the movies view

Comment (RamonaN) [reply frontend/index.html]: i need background on the body, i don't know how i can do it from movies ..

Comment (lightforsin) [reply frontend/index.html]: It's ok like this in the end, the homeCtrl should be here and not on the navbar.html which is a child of the parent template

**Pull request merged**